### ISSUE TITLE
atPath is inconsistent

### ISSUE THREAD

**Issue by pranayrauthu** [ryanfitzer/nmsp#2]
![image](https://user-images.githubusercontent.com/22414856/43563890-c04f0286-9641-11e8-94cf-8bdb1244dfa5.png)

![image](https://user-images.githubusercontent.com/22414856/43563967-19daf09e-9642-11e8-9c9f-14a8315180ae.png)

It would be better if nmsp exposes a function which accepts array of path.

**Comment by ryanfitzer** [comment]
> It would be better if nmsp exposes a function which accepts array of path.

I like the idea. Makes no sense to me why someone would use object notation in a property name, but I've definitely seen it. I'll add the option for `.atPath()` to take a string or an array:

```
nmsp.atPath( 'a.b.c.d', src );
```
Or:

```
nmsp.atPath( [ 'a', 'b', 'c', 'd' ], src );
```
 
When the path is a string, I'll split at `.` characters, but not when path is an array.

**Comment by ryanfitzer** [comment]
@pranayrauthu Just published v1.1.0 with support for paths as arrays. Thanks for the suggestion.
